### PR TITLE
[PAY-239] Add native supporter/supporting user lists

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds native supporter/supporting user lists
Adds supporting/supporter sections to user-list-item

image

### CODE REVIEW

File: packages/mobile/src/screens/user-list-screen/Tip.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export const Tip = (props: TipProps) => {
+  const { amount } = props
+  const styles = useStyles()
+  const { neutralLight4 } = useThemeColors()
+
+  return (
+    <View style={styles.root}>
+      <IconTip fill={neutralLight4} height={14} width={14} />
+      <Text color='neutralLight4'>
```
Comment (rickyrombo) [inline packages/mobile/src/screens/user-list-screen/Tip.tsx:28]: Does it make sense to wrap like this or should we just have `color='neutralLight4'` twice

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-239-supporter-user-lists

File: packages/mobile/src/screens/user-list-screen/UserListItem.tsx [hunk 2]
```diff
@@ -63,11 +66,12 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
 }))
 
 type UserListItemProps = {
+  tag: string
```
Comment (rickyrombo) [inline packages/mobile/src/screens/user-list-screen/UserListItem.tsx:69]: nit: made the same comment in @sddioulde 's PR - `tag` feels overloaded as a term, is there something else we could use? Also, should this maybe be an enum?

**Review by rickyrombo — APPROVED** [review]

File: packages/mobile/src/screens/user-list-screen/UserListItem.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/user-list-screen/UserListItem.tsx:69]: yes to all the above, ideally we have a better prop name, and then also define these tags in a common file. im down to look into that moving forward, just don't want to slow down this work for that large refactor? https://linear.app/audius/issue/C-541/consolidate-user-list-tags-and-rename-props

File: packages/mobile/src/screens/user-list-screen/Tip.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/user-list-screen/Tip.tsx:28]: i was trying something new out, but agreed it seems quite redundant here!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-239-supporter-user-lists

**Pull request merged**